### ISSUE TITLE
all are nan 

### ISSUE THREAD

**Issue by pujianjian** [Bohao-Lee/CME#11]
Hello, I follow your process, but during the training, after a while, all are nan . I'm very upset
![1629353991(1)](https://user-images.githubusercontent.com/48307115/130018035-3d6616c6-ec56-4831-bfb2-625520eafe70.png)

**Comment by Bohao-Lee** [comment]
> Hello, I follow your process, but during the training, after a while, all are nan . I'm very upset
> ![1629353991(1)](https://user-images.githubusercontent.com/48307115/130018035-3d6616c6-ec56-4831-bfb2-625520eafe70.png)
I have met this bug before. Maybe different GPU may causes this bug. You can modify cfg/darknet_dynamic.cfg
> steps=-1,500,40000,60000   
> scales=0.1,10,.1,.1 
to 
> steps=-1,500,40000,60000   
> scales=0.01,100,.1,.1 
can solve this bug.

**Comment by The-Hulk-007** [comment]
At the beginning, the learning rate == 0.001, and at the beginning of training, proposals == 50000. After a while, proposals == 0, and then "nan" appeared. I am so sad

Later, I modified batchsize="32", learning rate =0.0003, and just started training proposals==50000. After a little longer than last time, proposals==0 appeared, and "nan" did not appear.


一开始学习率==0.001，刚开始训练proposals==50000，过一会proposals==0，然后出现 “nan”。我好伤心

后面，我修改了batchsize="32"，learning rate =0.0003，刚开始训练proposals==50000，过了比上次更久一点，出现proposals==0，没有出现“nan”
![start](https://github.com/Bohao-Lee/CME/assets/56480454/ec4a259f-ea50-438a-92b2-06a33fdccc02)
![as time increases](https://github.com/Bohao-Lee/CME/assets/56480454/805c8e1a-2970-4cc5-af5f-6dc89c083a66)


![sad](https://github.com/Bohao-Lee/CME/assets/56480454/10d36565-547d-47bf-8cd2-e31198efd9dd)
help，being Very impatient
